rpc/rpc.go

```diff
@@ -7,7 +7,7 @@ import (
 	"sync"
 	"time"
 
-	"zombiezen.com/go/capnproto2"
+	capnp "zombiezen.com/go/capnproto2"
```

Comment:
nit: I don't think these explicit identifiers are needed. Can you help me understand why we're adding them here?

Reply:
My editor automatically does this when the last path component doesn't match the package name.

I've personally found it helpful on occasion, but I'm happy to change it.